src/group_theory/subsemigroup/center.lean

```diff
@@ -147,7 +145,7 @@ end
 section
 variables (M) [comm_semigroup M]
 
-@[to_additive, simp] lemma center_eq_top : center M = ⊤ :=
+@[simp] lemma center_eq_top : center M = ⊤ :=
```

Comment:
What's wrong with these `to_additive`s?

Reply:
I definitely didn't take them out explicitly. I think it may have been the result of them being updated in master, but I failed to merge correctly. It's been 2 months, so I don't remember. In any case, I've put them back in.